packages/react-composites/src/composites/common/AvatarPersona.tsx

```diff
@@ -77,7 +77,11 @@ export const AvatarPersona = (props: AvatarPersonaProps): JSX.Element => {
 
   useEffect(() => {
     (async () => {
-      if (dataProvider && userId) {
+      if (!userId) {
+        setData(undefined);
```

Comment:
We should set persona data to undefined so that custom avatar is undone when userId changes to undefined

Reply:
Under which case userId would be undefined? Is it when transferring the call but not connected yet? If so, why would the component still use the same old instance? Shouldn't that be a new instance of component + participant with undefined userId?

Reply:
Yes you are right. The transfer target participant should not have a userId that is undefined. Reverting these changes.